### ISSUE TITLE
Throws on using Autoprefixer

### ISSUE THREAD

**Issue by Antontelesh** [cssnext/cssnext#131]
Since Autoprefixer has been updated to v5.2, the function

```
autoprefixer: function(options) {
    return require("autoprefixer-core")(options).postcss
}
```

returns `undefined`. It then throws when using inside postcss.

I've raised an issue in autoprefixer repository: https://github.com/postcss/autoprefixer/issues/454
but it has been rejected.

**Comment by blittle** [comment]
This is killing us right now. Our entire project does not compile with webpack (using cssnext-loader) while this is broken. I fix it with npm shrinkwrap and forcing autoprefixer version to 5.1.11.

Feel super sick it took me so long to figure out this was the problem (happened mid large refactor).

**Comment by MoOx** [comment]
Sorry for that. Will push a fix really soon.

**Comment by stopsatgreen** [comment]
Can confirm this problem; I disabled autoprefixer in `gulp-cssnext` and the problem went away.

**Comment by MoOx** [comment]
Give me 20min. I am releasing the new cssnext website right now :)

**Comment by MoOx** [comment]
Fix released as 1.5.2

**Comment by blittle** [comment]
Thank you @MoOx !
